macros/ToolsSidebar.ejs

```diff
@@ -613,61 +613,76 @@ const text = mdn.localStringMap({
 
 <section class="Quick_links" id="Quick_Links">
   <ol>
-    <li><a href="#"><%=text["Core_Tools"]%></a>
-      <ol>
-        <li><a href="<%=baseURL%>Page_Inspector"><%=text["Page_Inspector"]%></a></li>
-        <li><a href="<%=baseURL%>Web_Console"><%=text["Web_Console"]%></a></li>
-        <li><a href="<%=baseURL%>Debugger"><%=text["JavaScript_Debugger"]%></a></li>
-        <li><a href="<%=baseURL%>Network_Monitor"><%=text["Network_Monitor"]%></a></li>
-        <li><a href="<%=baseURL%>Performance"><%=text["Performance"]%></a></li>
-        <li><a href="<%=baseURL%>Responsive_Design_Mode"><%=text["Responsive_Design_Mode"]%></a></li>
-        <li><a href="<%=baseURL%>Tips"><%=text["Tips"]%></a></li>
-      </ol>
+    <li class="toggle">
```

Comment:
The ``{{ToolsSidebar}}`` was converted correctly 👍 as seen on https://developer.mozilla.org/en-US/docs/Tools/Web_Console . There's some local weirdness because the macro has ``https://developer.mozilla.org`` hard-coded into some links, which make them looks like external links in local dev. This should be removed, but it was in the old version, so out-of-scope for this PR.

<img width="467" alt="pr 789 toolssiderbar" src="https://user-images.githubusercontent.com/286017/45395131-3460a300-b5fa-11e8-94d5-d1cb00178842.png">